core/appeal/service.go

```diff
@@ -438,6 +469,21 @@ func (s *Service) MakeAction(ctx context.Context, approvalAction domain.Approval
 						},
 					},
 				})
+
+				if appeal.AccountType == domain.DefaultAppealAccountType && appeal.AccountID != appeal.CreatedBy {
+					notifications = append(notifications, domain.Notification{
+						User: appeal.AccountID,
+						Message: domain.NotificationMessage{
+							Type: domain.NotificationTypeAppealApprovedForOther,
+							Variables: map[string]interface{}{
+								"resource_name": fmt.Sprintf("%s (%s: %s)", appeal.Resource.Name, appeal.Resource.ProviderType, appeal.Resource.URN),
+								"role":          appeal.Role,
+								"account_id":    appeal.AccountID,
+								"requestor":     appeal.CreatedBy,
```

Comment:
Can you also add the `appeal_id` here? If one wants to modify the message in the config and show `appeal_id` also in the notification, it will be helpful.

Reply:
done